### PR TITLE
create login page

### PR DESCRIPTION
**Pull request by Lamonaga**
create login page

### CODE REVIEW

File: src/components/UI/Input/index.tsx [hunk 1]
```diff
@@ -1,32 +1,35 @@
 import { memo } from 'react';
 import styled, { DefaultTheme, StyledComponentProps } from 'styled-components';
+import { forwardRef } from 'react';
```
Comment (Ivlay) [inline src/components/UI/Input/index.tsx]: remove duplicate imports

File: src/pages/login.tsx [hunk 2]
```diff
@@ -1,25 +1,61 @@
 import { NextPage } from 'next';
 import { useMutation } from '@apollo/client';
-import { Button } from 'src/components/UI';
+import { Input } from 'src/components/UI';
 import { LOGIN } from 'graphql/Post';
+import { useForm, SubmitHandler } from 'react-hook-form';
```
Comment (Ivlay) [inline src/pages/login.tsx]: import order

File: src/pages/login.tsx [hunk 3]
```diff
@@ -1,25 +1,61 @@
...
+
+interface IFormInput {
+  login: string;
+  password: string;
+}
 
 const Login: NextPage = () => {
+  const INPUTS = [
```
Comment (Ivlay) [inline src/pages/login.tsx]: move to constants

**Comment by Lamonaga** [comment]
asd

**Comment by Lamonaga** [comment]
123

File: src/components/_pages/login/constants/index.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { Input } from "src/types";
+
```
Comment (Ivlay) [inline src/components/_pages/login/constants/index.ts]: remove extra space

File: src/pages/login.tsx [hunk 5]
```diff
@@ -1,63 +1,13 @@
...
-              type={itemInput.type}
-              {...register(itemInput.name, { required: true })}
-            />
-          );
-        })}
-        {errors.username || errors.password ? <span>This field is required</span> : <></>}
-        <Input type="submit"/>
-      </form>
-    </div>
+    <>
```
Comment (Ivlay) [inline src/pages/login.tsx]: remove unused fragment

File: src/types/index.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { RegisterOptions } from 'react-hook-form';
+export interface Input {
```
Comment (Ivlay) [inline src/types/index.ts:3]: add space

File: src/components/_pages/login/constants/index.ts [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import { Input } from "src/types";
```
Comment (Ivlay) [inline src/components/_pages/login/constants/index.ts]: use single quotes

**Pull request merged**